test/src/e2e_vm_tests/test_programs/should_pass/stdlib/u256_test/test.toml

```diff
@@ -0,0 +1,3 @@
+category = "run"
+expected_result = { action = "return", value = 1 }
```

Comment:
I may be misremembering but why is a bool (true has the value of 1) being treated as a 1 here? Should this be a bool or is this fine? I guess it is fine

Reply:
The output here is untyped. True is 1 and false is 0 in the VM